README.md

```diff
@@ -132,6 +132,98 @@ In order to support atomify turtles all the way down, you can also specify your
 
 For detailed information on configuring Rework plugins in package.json see the [relevant section](https://github.com/Techwraith/atomify-css#packagejson-config) of the atomify-css README.
 
+## Mini Tutorial
+### How to build a re-usable component
+(__Warning:__ _The following example is only interesting from a technical and not at all from an esthetical perspective)_
```

Comment:
I like atomify a lot. Is there already a namespace on npm?
Is there a convention for namespaces?
`ecosystem:atomify` and `atomify-xy`? Maybe npm can even be scraped for those tags and list all "atomify components", because once many people create atomify components, that really creates value for everyone in this "new ecosystem".

The only thing i believe is really important is to maybe PREFIX every class with the component name (across html, js and css) to totally break css cascading (and even append the version number) to enable multiple, let's say "buttons" of different versions without having a conflict in styling.

Because once, people really start mix and matching tens and hundreds of components, there needs to be something to prevent css cascading from messing up the styling.

I'm currently investigating [eq](https://github.com/snugug/eq.js) and similar libraries to make up a standard of responsiveness based on the parent component. Has anyone experience with this already?
I also would love to see some feedback about the BEM naming convention i'm currently trying to use, which is
```
ComponentName
ComponentName--MODNAME_modValue
ComponentName__elementStyledByComponent
ComponentName__SubComponentName
ComponentName-path-to-object-value-to-insert-here
```
which would compile to version suffixed names
```
ComponentName0-0-1
ComponentName0-0-1--MODNAME_modValue
ComponentName0-0-1__elementStyledByComponent
ComponentName0-0-1__SubComponentName0-3-8
ComponentName0-0-1-path-to-object-value-to-insert-here
```